### PR TITLE
Version 0.0.3

### LINKED ISSUE #5: Add CI for Gem
### Feature Categories:

- [x] Other: Continuous Integration

**Describe the feature/Bug your gonna work on**
- Integrate Travis CI for gem. 
- Linked Deployment to RubyGems

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #6: Modify Gem to Return Objects on method call
### Feature Categories:

- [x] Other: Refactor/Enhancement

**Describe the feature/Bug your gonna work on**
- Refactor/add code to return Objects on method call for API endpoints. 
- This would have a little bit more "real-world" application. 

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #8: Add RDoc documentation for Som Timer gem
### Feature Categories:

- [x] Documentation

**Describe the feature/Bug your gonna work on**
- Add RDoc documentation for Gem
- [Helpful tutorial on this](https://guides.rubygems.org/make-your-own-gem/) 

**Additional context**
Add any other context or screenshots about the feature.
- gem docs are super important!

### PR DESCRIPTION
**Pull request by sienna-kopf**
# SOM-Timer Ruby Gem Pull Request Form

## Description
Please include a summary of the change.
- refactor gem to return Ruby Objects 
- refactor/reorganize code to leverage use of facades and poros. 
- TDD everything -- 100% coverage.
- Built and pushed to Ruby Gems [Som Timer Gem Version 0.0.3](https://rubygems.org/search?utf8=%E2%9C%93&query=som-timer)
- Add continuous integration with Travis CI and hooks for automatic deploy to Ruby Gems when passing
- Documented all methods of main Som Timer class using RDoc.  14.04% documented so far!
- Updated ReadMe

## Setup/Usage
Include steps for setup/usage.
- See readme

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Has This Been Tested?
- [ ] No
- [x] Yes

Details:
Run test suite using:
```
bundle exec rspec
```

Open coverage report using:
```
open coverage/index.html
```

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs/pry sessions
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ $ irb
 > SomTimer.create_rest(<mood_rating_1>, <mood_rating_2>, <content_selected>, <focus_interval>, <rest_interval>)
 ```
 
+All endpoints return a Ruby Object or a collection or Ruby Objects for ease of use.
```
Comment (rwilliams659) [inline README.md]: Woohoo!

**Review by rwilliams659 — APPROVED** [review]
Hard for me to follow all of the code exactly, but I'm just impressed by how much of it there is & how thoroughly tested it appears to be. Great work!

**Review by Chulstro — APPROVED** [review]
Where did you go to figure all of this out? It is incredibly interesting

**Comment by sienna-kopf** [comment]
> Where did you go to figure all of this out? It is incredibly interesting

Google land! and pulling from old projects. It was a lot like the APIs we've built in the past. Configuring the .gemspec, the rdoc stuff, and actually knowing what to require was the hardest part.

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**